src/script/main/app.ts

```diff
@@ -387,9 +387,9 @@ export class App {
 
       eventRepository.setEventProcessMiddlewares([
         serviceMiddleware,
-        quotedMessageMiddleware,
         readReceiptMiddleware,
         eventStorageMiddleware,
+        quotedMessageMiddleware,
```

Comment:
We want to make sure the event is stored before processing quote updates. It's now possible to do it cleanly with the order of the middlwares